subsys/dfu/img_util/flash_img.c

```diff
@@ -83,3 +84,36 @@ int flash_img_init(struct flash_img_context *ctx)
 {
 	return flash_img_init_id(ctx, FLASH_AREA_IMAGE_SECONDARY);
 }
+
+#if defined(CONFIG_IMG_ENABLE_IMAGE_CHECK)
+int flash_img_check(struct flash_img_context *ctx,
+		    const struct flash_img_check *fic,
+		    uint8_t area_id)
+{
+	struct flash_area_check fac;
+	int rc;
+
+	if (!ctx || !fic) {
+		return -EFAULT;
```

Comment:
I think that you should be consistent with all other cases when you process `NULL` pointers an return `-EINVAL` here.

Reply:
I miss that! Thanks